modules/storages/app/common/storages/peripherals/nextcloud_connection_validator.rb

```diff
@@ -71,14 +71,23 @@ def audience_missing
       end
 
       def non_provisioned_user
-        return None() if @user.authentication_provider.present?
+        return None() if @user.identity_url.present?
 
         Some(ConnectionValidation.new(type: :warning,
                                       error_code: :oidc_non_provisioned_user,
                                       timestamp: Time.current,
                                       description: I18n.t("storages.health.connection_validation.oidc_non_provisioned_user")))
       end
 
+      def oidc_provisioned_user
```

Comment:
:yellow_circle: I think these methods are usually named after "what could be wrong here", but the user being OIDC provisioned means it's not wrong.

--> `not_oidc_provsioned_user`? `non_oidc_user`?

Not sure about a really good name for this, just noticed that the method had the name of the good case.